### DISCUSSION TITLE
Does WiringPi support the latest 64-bit Raspbian?

### DISCUSSION THREAD

**Discussion by regunakyle** [WiringPi/WiringPi#248] [Q&A] [answered]
Just installed the `armhf` version and got hit with `gpio: error while loading shared libraries: libcrypt.so.1: cannot open shared object file: No such file or directory` when I ran `gpio`. 

Some online articles suggest this is due to my 64 bit OS. Is this true? 

Note: I am using a Raspberry Pi 4B

**Comment by wangyujin0831** [comment, marked as answer]
maybe you should download from Release and select arm64 pakg .
when you finish pakg install manually, you can run gpio readall to check.
usually its working.
my env:
ubuntu arm 64
Raspberry Pi 4B
